src/scss/ui/_tag.scss

```diff
@@ -0,0 +1,68 @@
+.tag-container {
+    display: flex;
+    flex-wrap: wrap;
+
+    .tag {
+        background-color: $tag-bg-color;
+        height: 23px;
+        border: 1px solid gray;
+        padding: 0 .2rem;
+        margin: 0 1rem;
+        border-radius: 5px;
+    }
+
+    .cross-icon-tag {
+        display: flex;
+        align-items: center;
+        gap: $tag-gap;
+
+        .icon {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            padding: 0;
+            margin: 0;
+            height: 16px;
+            font-size: 14px;
+            transition: .5 all;
+
+            &:hover {
+                background-color: $tag-icon-hover-color;
+                cursor: pointer;
+            }
+        }
+    }
+
+    .number {
+        background: $tag-icon-hover-color;
+        height: 16px;
+    }
+
+    .status-tag {
+        display: flex;
+        align-items: center;
+        gap: $tag-gap;
+
+        .status-circle {
+
+            height: 8px;
+            width: 8px;
+            border-radius: 50%;
+            background-color: $tag-status-online;
+        }
+    }
+
+    .box-status-tag {
+        display: flex;
+        align-items: center;
+        gap: $tag-gap;
+
+        .box-status {
```

Comment:
we have `form-check` component and it should be used there instead of creating new elements https://preview.tabler.io/form-elements.html